src/ray/common/ray_config_def.h

```diff
@@ -318,6 +318,15 @@ RAY_CONFIG(uint32_t, task_retry_delay_ms, 0)
 /// Duration to wait between retrying to kill a task.
 RAY_CONFIG(uint32_t, cancellation_retry_ms, 2000)
 
+/// Whether to start a background thread to import Python dependencies eagerly.
+/// When set to false, Python dependencies will still be imported, only when
+/// they are needed.
+RAY_CONFIG(bool, start_python_importer_thread, true)
+
+/// Determines if forking in Ray actors / tasks are supported.
+/// Note that this only enables forking in workers, but not drivers.
+RAY_CONFIG(bool, support_fork, false)
```

Comment:
The same, why not turn on this support by default?

Reply:
This flag sets GRPC flags that do not apply to all OS, and may reduce performance. So it should only be enabled when needed.

Reply:
Note that `GRPC_ENABLE_FORK_SUPPORT` can cause hangs, so definitely don't set this true by default :-)

Let me know in #openai-anyscale if you need more details